### PR TITLE
feature: global nav design review [NP-1026]

### PR DESCRIPTION
**Pull request by davidsauntson**
Brings the 'more' drop down in global navigation in line with the designs:
 - on `sm`, the drop down is full width
 - on `md` and `lg`, the drop down is aligned to the right of the viewport
 - spacing and padding corrected as per designs

### CODE REVIEW

File: scss/2-tools/_interactive-elements.scss [hunk 1]
```diff
@@ -15,3 +15,40 @@
     @include cads-interactive-focus();
   }
 }
+
+@mixin cads-nav-link-focus() {
+  background-color: $cads-palette__white;
```
Comment (baseonmars) [inline scss/2-tools/_interactive-elements.scss:20]: can these be swapped out for a `$cads-language` variable to make theming a bit easier? Perhaps out of scope.

File: scss/2-tools/_interactive-elements.scss [hunk 2]
```diff
@@ -15,3 +15,40 @@
...
+  color: $cads-language__brand-primary;
+  border: $cads-border-width-medium solid $cads-palette__heritage-yellow;
+  outline: none;
+}
+
+@mixin cads-nav-link() {
+  font-size: 1rem;
```
Comment (baseonmars) [inline scss/2-tools/_interactive-elements.scss:27]: Some opportunity to use more vars here and on line-height?

File: scss/6-components/_navigation.scss [hunk 3]
```diff
@@ -80,25 +55,28 @@
...
       display: block;
       content: '';
       border-top: $cads-border-width-small solid $cads-palette__white;
       position: relative;
-      left: $cads-spacing-4;
-      width: calc(100% - #{2 * $cads-spacing-4});
+      // take focus border into account
+      left: cads-rem-to-px($cads-spacing-4) + $cads-border-width-medium;
+      width: calc(
+        100% - #{2 * $cads-spacing-4} - #{2 * $cads-border-width-medium}
```
Comment (baseonmars) [inline scss/6-components/_navigation.scss:67]: TIL

File: src/ts/greedy-nav/Config.ts [hunk 4]
```diff
@@ -69,7 +69,7 @@ const defaultConfig: Config = {
   navDropdownBreakpointLabel: 'menu',
   breakPoint: 0,
   throttleDelay: 50,
-  offsetPixels: 0,
+  offsetPixels: -2,
```
Comment (baseonmars) [inline src/ts/greedy-nav/Config.ts:72]: Why does it need this?

**Review by baseonmars — COMMENTED** [review]
Gonna check in browser - but a few minor comments for you to chew on.

**Review by baseonmars — APPROVED** [review]
checked in browser :+1:

File: scss/2-tools/_interactive-elements.scss (hunk 1 above)
Comment (davidsauntson) [reply scss/2-tools/_interactive-elements.scss:20]: `$cads-language__brand-primary-background`

File: src/ts/greedy-nav/Config.ts (hunk 4 above)
Comment (davidsauntson) [reply src/ts/greedy-nav/Config.ts:77]: added comment in config setting

**Pull request merged**